config/application.rb

```diff
@@ -1,31 +1,32 @@
-require File.expand_path('../boot', __FILE__)
+require_relative "boot"
 
-require 'rails/all'
+require "rails/all"
 
-if defined?(Bundler)
-  Bundler.require *Rails.groups(:assets => %w(development test))
-end
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
 
 module IndyhackersRails
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.1
+
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
+    #
+    # config.time_zone = "Central Time (US & Canada)"
+    # config.eager_load_paths << Rails.root.join("extras")
+
     config.app_generators do |g|
       g.template_engine :haml
       g.test_framework :rspec, :fixture => true, :views => false
       g.fixture_replacement :factory_bot, :dir => "spec/factories"
     end
 
-    config.active_record.whitelist_attributes
     config.encoding = "utf-8"
-    config.filter_parameters += [:password, :password_confirmation]
     config.assets.enabled = true
-    config.assets.precompile += [
```

Comment:
Moved to `config/initializers/assets.rb` with the rest of the asset configuration.